src/entities/contact-list/contact-list.ts

```diff
@@ -58,6 +58,15 @@ export interface ContactListRawPayload {
   readonly static_entries?: ContactListStaticEntryRawPayload[]
   readonly labels?: object
   readonly parent?: string
+  readonly external_list_reference_id?: string
+  readonly external_list_type?: 'list' | 'segment' | null
+  readonly broker?: string
+  readonly source_type?: string
```

Comment:
are there some hard coded strings that we can expect from broker, source_type, source_api?